src/linker/cli.py

```diff
@@ -24,13 +23,19 @@ def linker():
 
 
 @linker.command()
-@click.argument(
-    "pipeline_specification",
+@click.option(
+    "-p",
+    "--pipeline-specification",
+    required=True,
     type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+    help="The path to the pipeline specification yaml file.",
 )
-@click.argument(
-    "input_data",
+@click.option(
+    "-i",
+    "--input-data",
+    required=True,
```

Comment:
Should these be arguments if they are required?

Reply:
No, we agreed on "required options" b/c they don't need to be ordered and support --help strings